requirements.txt

```diff
@@ -5,3 +5,4 @@ vgamepad==0.0.8
 pyttsx3==2.90
 pyyaml==6.0
 tqdm==4.64.1
+jsonmerge==1.9.0
```

Comment:
Whoever merges this needs to announce that there are new reqs, so we need to run `pip install -r requirements.txt` to install the new reqs.